`Export` and `SortedExport` commands must be resilient to the corrupted state

If the state is corrupted, the export commands will be failing. 

Sorted:
```
	at com.hedera.statevalidation.ExportCommand.run(ExportCommand.java:71)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2045)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2469)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2423)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2277)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2425)
	at picocli.CommandLine.execute(CommandLine.java:2174)
	at com.hedera.statevalidation.StateOperatorCommand.main(StateOperatorCommand.java:36)
Caused by: java.lang.NullPointerException: Cannot invoke "com.swirlds.virtualmap.datasource.VirtualLeafBytes.keyBytes()" because "leafRecord" is null
	at com.hedera.statevalidation.exporters.SortedJsonExporter.lambda$collectKeys$5(SortedJsonExporter.java:138)
```

Unsorted:
```
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "com.swirlds.virtualmap.datasource.VirtualLeafBytes.keyBytes()" because "leafRecord" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
	at java.base/java.util.concurrent.ThreadPerTaskExecutor$TaskRunner.run(ThreadPerTaskExecutor.java:314)
	at java.base/java.lang.VirtualThread.run(VirtualThread.java:309)
Caused by: java.lang.NullPointerException: Cannot invoke "com.swirlds.virtualmap.datasource.VirtualLeafBytes.keyBytes()" because "leafRecord" is null
	at com.hedera.statevalidation.exporters.JsonExporter.processRange(JsonExporter.java:117)
	at com.hedera.statevalidation.exporters.JsonExporter.lambda$traverseVmInParallel$0(JsonExporter.java:105)
```

When these issues occur, they should be reported but not fail the entire export

--- a/hedera-state-validator/src/main/java/com/hedera/statevalidation/exporter/JsonExporter.java
+++ b/hedera-state-validator/src/main/java/com/hedera/statevalidation/exporter/JsonExporter.java
@@ -150,7 +150,16 @@ public class JsonExporter {
         boolean emptyFile = true;
         try (BufferedWriter writer = new BufferedWriter(new FileWriter(file))) {
             for (long path = start; path <= end; path++) {
-                VirtualLeafBytes leafRecord = vm.getRecords().findLeafRecord(path);
+                VirtualLeafBytes leafRecord = null;
+                try {
+                    leafRecord = vm.getRecords().findLeafRecord(path);
+                } catch (final Exception e) {
+                    log.error("Unexpected error while finding leaf record by path", e);
+                }
+                if (leafRecord == null) {
+                    log.error("No leaf record for path {}", path);
+                    continue;
+                }
                 final Bytes keyBytes = leafRecord.keyBytes();
                 final Bytes valueBytes = leafRecord.valueBytes();
                 final StateKey stateKey;

--- a/hedera-state-validator/src/main/java/com/hedera/statevalidation/exporter/SortedJsonExporter.java
+++ b/hedera-state-validator/src/main/java/com/hedera/statevalidation/exporter/SortedJsonExporter.java
@@ -176,7 +176,16 @@ public class SortedJsonExporter {
         }
 
         LongStream.range(firstLeafPath, lastLeafPath + 1).parallel().forEach(path -> {
-            final VirtualLeafBytes leafRecord = vm.getRecords().findLeafRecord(path);
+            VirtualLeafBytes leafRecord = null;
+            try {
+                leafRecord = vm.getRecords().findLeafRecord(path);
+            } catch (final Exception e) {
+                log.error("Unexpected error while finding leaf record by path", e);
+            }
+            if (leafRecord == null) {
+                log.error("No leaf record for path {}", path);
+                return;
+            }
             final Bytes keyBytes = leafRecord.keyBytes();
             final ReadableSequentialData keyData = keyBytes.toReadableSequentialData();
             final int tag = keyData.readVarInt(false);

